### ISSUE TITLE
Split Accounts domain into multiple domains

### ISSUE THREAD

**Issue by amos-kibet** [3Stones-io/omedis#418]
Our `Accounts` domain is getting pretty big. It's time to start splitting it into multiple domains.

**Comment by almirsarajcic** [comment]
I propose we do it this way:
- Accounts
  - Organisation
  - Token
  - User
- AccessRights
  - AccessRight
- Groups
  - Group
  - GroupMembership
- Invitations
  - Invitation
  - InviteeNotifier (ex UserNotifier)
- Projects
  - Project
- TimeTracking
  - Activity
  - Event

I wanted us to put `Group` and `GroupMembership` together with `AccessRight`, but groups aren't only used for permissions, but for time tracking as well. For the `Project`, this could go into `TimeTracking`, but pretty soon we'll be introducing some kind of forms which won't be related to time tracking.

**Comment by almirsarajcic** [comment]
@kagure-nyakio, should we consider this done after #450 is merged or should we wait for #419?

**Comment by kagure-nyakio** [comment]
After #450, we can consider #419 as a separate task

**Comment by almirsarajcic** [comment]
Divided by domains into the following tasks:
- #445 
- #446 
- #447 
- #448 
- #449 
- #450

We'll work on code interfaces (#419) later.
